### PR TITLE
Small doc update

### PR DESCRIPTION
**Pull request by evadb**
As requested by @sidewalkballet , this PR adds some additional details to the google analytics document.

### CODE REVIEW

File: docs/google-analytics.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
 # Google Analytics
 
-This project includes a Google Analytics set up with custom events for CSV downloads and clicking Organization links, to track what data users are looking at/searching for. To make use of Google Analytics, you will have to create your own Google Analytics account and plug your new UA code in the `gtag('config')` function in `templates/includes/head.html`. Everything else should work automatically.
\ No newline at end of file
+This project includes a Google Analytics set up with custom events for CSV downloads and clicking Organization links, to track what data users are looking at/searching for. To make use of Google Analytics, you will have to create your own Google Analytics account and plug your new UA code in the `gtag('config')` function in `templates/includes/head.html`. Everything else should work automatically.
+
+Two custom events are included to help track how users are interacting with the product. 
+
+## CSV Download
+![alt text](download-csv.png "Example image showing download CSV events in Google Analytics")
```
Comment (buckley-w-david) [inline docs/google-analytics.md]: When we send off the docs for translations, will we need a french version of this image?

Comment (evadb) [reply docs/google-analytics.md]: There is no french google analytics

Comment (sidewalkballet) [reply docs/google-analytics.md]: Yes we will.

Comment (evadb) [reply docs/google-analytics.md]: Going to just delete the images

File: docs/google-analytics.md [hunk 2]
```diff
@@ -5,11 +5,9 @@ This project includes a Google Analytics set up with custom events for CSV downl
 Two custom events are included to help track how users are interacting with the product. 
 
 ## CSV Download
-![alt text](download-csv.png "Example image showing download CSV events in Google Analytics")
 
 There are two events for CSV downloads: full CSV & specific domain CSV. This are categorized under "Download" actions.
```
Comment (sidewalkballet) [inline docs/google-analytics.md]: Are they called "event actions"? Could be helpful to include how to get there (Behavior -> Events -> Top Events -> "Download").

**Pull request merged**